apps/docs/src/packages/headless.md

```diff
@@ -0,0 +1,58 @@
+---
+outline: [2, 3]
+---
+
+<script lang="ts" setup>
+import packageJson from "../../../../packages/headless/package.json";
+</script>
+
+# @sit-onyx/headless
+
+::: warning Work in progress / Active development
+This library is currently in early / active development.
+:::
+
+{{ packageJson.description }}.
+
+Inspired by [Melt UI](https://melt-ui.com).
+
+## Installation
+
+<!--
+Make sure that this chapter is kept up to date with installation steps in
+packages/headless/README.md file.
+ -->
+
+Install the npm package with your corresponding package manager:
+
+::: code-group
+
+```sh [pnpm]
+pnpm add sit-onyx/headless
+```
+
+```sh [npm]
+npm install sit-onyx/headless
+```
+
+```sh [yarn]
+yarn install sit-onyx/headless
+```
+
+:::
+
+## Composables
+
+### useComboBox
```

Comment:
do we really want to show the usage of all the components we'll build or shouldn't we name this more like "Example usage"

Reply:
I think we should show the basic usage yes, esspecially because other projects will also use them (e.g. ODS) so they need to know which composables exist and how they are used :)